xml/System.Data.Common/DbBatchCommandCollection.xml

```diff
@@ -127,10 +127,10 @@
         <Parameter Name="item" Type="System.Data.Common.DbBatchCommand" />
       </Parameters>
       <Docs>
-        <param name="item">The object to locate in the <see cref="T:System.Collections.Generic.ICollection`1" />.</param>
-        <summary>Determines whether the <see cref="T:System.Collections.Generic.ICollection`1" /> contains a specific value.</summary>
+        <param name="value">The <see cref="P:System.Data.Common.DbBatchCommand" /> to look for in the collection.</param>
```

Comment:
This parameter also got renamed mistakenly, and throughout this file.

Reply:
Sorry about this. Just to confirm - do you need me to do anything here or is it all fixed in #7909?

Reply:
#7909 corrects the parameter names, but unfortunately results in lost text descriptions. So I've logged #7910 to track bringing those comments back.